homeassistant/components/whirlpool/strings.json

```diff
@@ -122,6 +122,99 @@
       },
       "end_time": {
         "name": "End time"
+      },
+      "oven_state": {
+        "name": "Oven state",
```

Comment:
```suggestion
```
Since this is the main state of a single-cavity oven, I think we can remove the name so that it gets the device name instead: https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations